libnetwork/internal/kvstore/boltdb/boltdb.go

```diff
@@ -20,6 +21,12 @@ var (
 
 const filePerm = 0o644
 
+// Config contains the options for a BoltDB bucket.
+type Config struct {
+	ConnectionTimeout time.Duration
```

Comment:
Interesting; didn't an earlier commit remove an option to set the timeout? Should we have wired it up instead of removing? Looks like we hard-code it to 1 minute; is that something that should be configurable?

If we don't need it to be configurable, we could even remove this field

Even further; it looks like `Bucket` is also hardcoded to `libnetwork` 

❓ Do we need this Config struct _at all_ ? All options are hard-coded!